Airbitz/airbitz/src/main/java/com/airbitz/fragments/login/LandingFragment.java

```diff
@@ -266,6 +266,12 @@ public void onClick(View view) {
             }
         });
 
+        if (R.bool.auto_upload_logs == 1) {
+            AirbitzCore.logi("Auto-uploading logs from Landing screen");
+            UploadLogAlert uploadLogAlert = new UploadLogAlert(mActivity);
+            uploadLogAlert.showUploadLogAlert();
```

Comment:
This can't show an alert. It needs to immediately upload the logs. This is to help users who have an app crash upon app launch. They may never see the alert in the first place. Also, if possible, we need this to happen immediately after the AirbitzCore object is initialized so we can get the logs as soon as possible before things have a chance to crash.

Reply:
Gotcha, I misunderstood. On it!